lib/perl/Genome/Site/TGI/Extension/Sys.pm

```diff
@@ -128,6 +123,15 @@ sub _bsub_and_wait_for_completion__validate_args {
     return(\%special_args, %bsub_args);
 }
 
+sub _bsub_and_wait_for_completion__wait_on_jobs_with_lsf_dependency {
+    my ($class, $job_ids, $bsub_args) = @_;
+    local $ENV{LSF_NIOS_JOBSTATUS_INTERVAL} = 1;
+    $class->bsub(%$bsub_args,
```

Comment:
By "work won't proceed", I mean the original process calling bsub_and_wait_for_completion() will stay blocked there because the blocking process (the new LSF job dependent on all the worker jobs) isn't getting scheduled.  We've had this problem in the past where work gets deadlocked because an LSF job is waiting on another LSF job with equal or lower scheduling priority.